Subject: m68k: Replace `Linux/68k' by `Linux/m68k'

m68k: Replace `Linux/68k' by `Linux/m68k'

We switched to `Linux/m68k' a very long time ago

From: Geert Uytterhoeven <geert@linux-m68k.org>
Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 CREDITS                               |    4 ++--
 Documentation/devices.txt             |   10 +++++-----
 Documentation/m68k/kernel-options.txt |    2 +-
 arch/m68k/Kconfig                     |    2 +-
 arch/m68k/ifpsp060/iskeleton.S        |    6 +++---
 arch/m68k/ifpsp060/os.S               |    4 ++--
 arch/m68k/kernel/head.S               |    2 +-
 arch/m68knommu/Kconfig                |    2 +-
 drivers/net/a2065.c                   |    4 ++--
 drivers/net/a2065.h                   |    4 ++--
 drivers/net/apne.c                    |    2 +-
 include/asm-m68knommu/bootstd.h       |    2 +-
 include/linux/major.h                 |    2 +-
 13 files changed, 23 insertions(+), 23 deletions(-)

--- a/CREDITS
+++ b/CREDITS
@@ -174,7 +174,7 @@ N: Ralf Baechle
 E: ralf@gnu.org
 P: 1024/AF7B30C1 CF 97 C2 CC 6D AE A7 FE  C8 BA 9C FC 88 DE 32 C3
 D: Linux/MIPS port
-D: Linux/68k hacker
+D: Linux/m68k hacker
 S: Hauptstrasse 19
 S: 79837 St. Blasien
 S: Germany
@@ -2123,7 +2123,7 @@ D: dv1394
 
 N: Hamish Macdonald
 E: hamishm@lucent.com
-D: Linux/68k port
+D: Linux/m68k port
 S: 32 Clydesdale Avenue
 S: Kanata, Ontario
 S: Canada K2M-2G7
--- a/Documentation/devices.txt
+++ b/Documentation/devices.txt
@@ -19,8 +19,8 @@ The LaTeX version of this document is no
 This document is included by reference into the Filesystem Hierarchy
 Standard (FHS).	 The FHS is available from http://www.pathname.com/fhs/.
 
-Allocations marked (68k/Amiga) apply to Linux/68k on the Amiga
-platform only.	Allocations marked (68k/Atari) apply to Linux/68k on
+Allocations marked (m68k/Amiga) apply to Linux/m68k on the Amiga
+platform only.	Allocations marked (m68k/Atari) apply to Linux/m68k on
 the Atari platform only.
 
 The symbol {2.6} means the allocation is obsolete and scheduled for
@@ -328,7 +328,7 @@ Your cooperation is appreciated.
 		  2 = /dev/inportbm	Microsoft Inport bus mouse
 		  3 = /dev/atibm	ATI XL bus mouse
 		  4 = /dev/jbm		J-mouse
-		  4 = /dev/amigamouse	Amiga mouse (68k/Amiga)
+		  4 = /dev/amigamouse	Amiga mouse (m68k/Amiga)
 		  5 = /dev/atarimouse	Atari mouse
 		  6 = /dev/sunmouse	Sun mouse
 		  7 = /dev/amigamouse1	Second Amiga mouse
@@ -691,7 +691,7 @@ Your cooperation is appreciated.
 		  2 = /dev/staliomem2	Third Stallion card I/O memory
 		  3 = /dev/staliomem3	Fourth Stallion card I/O memory
 
- 28 char	Atari SLM ACSI laser printer (68k/Atari)
+ 28 char	Atari SLM ACSI laser printer (m68k/Atari)
 		  0 = /dev/slm0		First SLM laser printer
 		  1 = /dev/slm1		Second SLM laser printer
 		    ...
@@ -701,7 +701,7 @@ Your cooperation is appreciated.
 		  2 = /dev/sbpcd14	Panasonic CD-ROM controller 3 unit 2
 		  3 = /dev/sbpcd15	Panasonic CD-ROM controller 3 unit 3
 
- 28 block	ACSI disk (68k/Atari)
+ 28 block	ACSI disk (m68k/Atari)
 		  0 = /dev/ada		First ACSI disk whole disk
 		 16 = /dev/adb		Second ACSI disk whole disk
 		 32 = /dev/adc		Third ACSI disk whole disk
--- a/Documentation/m68k/kernel-options.txt
+++ b/Documentation/m68k/kernel-options.txt
@@ -852,7 +852,7 @@ use of this option is now highly unrecom
   Incorrect use can lead to unpredictable behavior, so please only use
 this option if you *know* what you are doing and have a reason to do
 so. In any case if you experience problems and need to use this
-option, please inform us about it by mailing to the Linux/68k kernel
+option, please inform us about it by mailing to the Linux/m68k kernel
 mailing list.
 
   The address mask set by this option specifies which addresses are
--- a/arch/m68k/Kconfig
+++ b/arch/m68k/Kconfig
@@ -52,7 +52,7 @@ config NO_IOPORT
 config NO_DMA
 	def_bool SUN3
 
-mainmenu "Linux/68k Kernel Configuration"
+mainmenu "Linux/m68k Kernel Configuration"
 
 source "init/Kconfig"
 
--- a/arch/m68k/ifpsp060/iskeleton.S
+++ b/arch/m68k/ifpsp060/iskeleton.S
@@ -65,7 +65,7 @@
 | To simply continue execution at the next instruction, just
 | do an "rte".
 |
-| Linux/68k: If returning to user space, check for needed reselections.
+| Linux/m68k: If returning to user space, check for needed reselections.
 
 	.global		_060_isp_done
 _060_isp_done:
@@ -87,7 +87,7 @@ _060_isp_done:
 | a CHK exception stack frame from the Unimplemented Integer Instrcution
 | stack frame and branches to this routine.
 |
-| Linux/68k: commented out test for tracing
+| Linux/m68k: commented out test for tracing
 
 	.global		_060_real_chk
 _060_real_chk:
@@ -127,7 +127,7 @@ real_chk_end:
 | then it create a Trace exception stack frame from the "chk" exception
 | stack frame and branches to the _real_trace() entry point.
 |
-| Linux/68k: commented out test for tracing
+| Linux/m68k: commented out test for tracing
 
 	.global		_060_real_divbyzero
 _060_real_divbyzero:
--- a/arch/m68k/ifpsp060/os.S
+++ b/arch/m68k/ifpsp060/os.S
@@ -65,7 +65,7 @@
 | The result is that Unix processes are allowed to sleep as a consequence
 | of a page fault during a _copyout.
 |
-| Linux/68k: The _060_[id]mem_{read,write}_{byte,word,long} functions
+| Linux/m68k: The _060_[id]mem_{read,write}_{byte,word,long} functions
 | (i.e. all the known length <= 4) are implemented by single moves
 | statements instead of (more expensive) copy{in,out} calls, if
 | working in user space
@@ -305,7 +305,7 @@ dmwls:	move.l		%d0,(%a0)		| store super 
 | Assumes that D0/D1/A0/A1 are scratch registers. The _copyin/_copyout
 | below assume that the SFC/DFC have been set previously.
 |
-| Linux/68k: These are basically non-inlined versions of
+| Linux/m68k: These are basically non-inlined versions of
 | memcpy_{to,from}fs, but without long-transfer optimization
 | Note: Assumed that SFC/DFC are pointing correctly to user data
 | space... Should be right, or are there any exceptions?
--- a/arch/m68k/kernel/head.S
+++ b/arch/m68k/kernel/head.S
@@ -1,7 +1,7 @@
 /* -*- mode: asm -*-
 **
 ** head.S -- This file contains the initial boot code for the
-**	     Linux/68k kernel.
+**	     Linux/m68k kernel.
 **
 ** Copyright 1993 by Hamish Macdonald
 **
--- a/arch/m68knommu/Kconfig
+++ b/arch/m68knommu/Kconfig
@@ -3,7 +3,7 @@
 # see Documentation/kbuild/kconfig-language.txt.
 #
 
-mainmenu "uClinux/68k (w/o MMU) Kernel Configuration"
+mainmenu "uClinux/m68k (w/o MMU) Kernel Configuration"
 
 config M68K
 	bool
--- a/drivers/net/a2065.c
+++ b/drivers/net/a2065.c
@@ -1,5 +1,5 @@
 /*
- * Amiga Linux/68k A2065 Ethernet Driver
+ * Amiga Linux/m68k A2065 Ethernet Driver
  *
  * (C) Copyright 1995-2003 by Geert Uytterhoeven <geert@linux-m68k.org>
  *
@@ -12,7 +12,7 @@
  *
  * This program is based on
  *
- *	ariadne.?:	Amiga Linux/68k Ariadne Ethernet Driver
+ *	ariadne.?:	Amiga Linux/m68k Ariadne Ethernet Driver
  *			(C) Copyright 1995 by Geert Uytterhoeven,
  *                                            Peter De Schrijver
  *
--- a/drivers/net/a2065.h
+++ b/drivers/net/a2065.h
@@ -1,5 +1,5 @@
 /*
- * Amiga Linux/68k A2065 Ethernet Driver
+ * Amiga Linux/m68k A2065 Ethernet Driver
  *
  * (C) Copyright 1995 by Geert Uytterhoeven <geert@linux-m68k.org>
  *
@@ -7,7 +7,7 @@
  *
  * This program is based on
  *
- *	ariadne.?:	Amiga Linux/68k Ariadne Ethernet Driver
+ *	ariadne.?:	Amiga Linux/m68k Ariadne Ethernet Driver
  *			(C) Copyright 1995 by Geert Uytterhoeven,
  *			Peter De Schrijver
  *
--- a/drivers/net/apne.c
+++ b/drivers/net/apne.c
@@ -1,5 +1,5 @@
 /*
- * Amiga Linux/68k 8390 based PCMCIA Ethernet Driver for the Amiga 1200
+ * Amiga Linux/m68k 8390 based PCMCIA Ethernet Driver for the Amiga 1200
  *
  * (C) Copyright 1997 Alain Malek
  *                    (Alain.Malek@cryogen.com)
--- a/include/asm-m68knommu/bootstd.h
+++ b/include/asm-m68knommu/bootstd.h
@@ -30,7 +30,7 @@
 #define __BN_flash_erase_range		19
 #define __BN_flash_write_range		20
 
-/* Calling conventions compatible to (uC)linux/68k
+/* Calling conventions compatible to (uC)linux/m68k
  * We use simmilar macros to call into the bootloader as for uClinux
  */
 
--- a/include/linux/major.h
+++ b/include/linux/major.h
@@ -53,7 +53,7 @@
 #define STL_SIOMEMMAJOR		28
 #define ACSI_MAJOR		28
 #define AZTECH_CDROM_MAJOR	29
-#define GRAPHDEV_MAJOR		29   /* SparcLinux & Linux/68k /dev/fb */
+#define GRAPHDEV_MAJOR		29   /* SparcLinux & Linux/m68k /dev/fb */
 #define CM206_CDROM_MAJOR	32
 #define IDE2_MAJOR		33
 #define IDE3_MAJOR		34
